launch_ros/launch_ros/descriptions/composable_lifecycle_node.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2025 Open Navigation LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for a description of a ComposableLifecycleNode."""
+
+from typing import List
+from typing import Optional
+
+import launch
+from launch.condition import Condition
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.substitution import Substitution
+# from launch.utilities import ensure_argument_type
+from launch.utilities import perform_substitutions
+from launch_ros.parameters_type import Parameters
+from launch_ros.parameters_type import SomeParameters
+from launch_ros.remap_rule_type import RemapRules
+from launch_ros.remap_rule_type import SomeRemapRules
+from launch_ros.utilities import LifecycleEventManager
+
+from .composable_node import ComposableNode
+
+
+class ComposableLifecycleNode(ComposableNode):
+    """Describes a lifecycle node that can be loaded into a container with other nodes."""
+
+    def __init__(
+        self, *,
+        package: SomeSubstitutionsType,
+        plugin: SomeSubstitutionsType,
+        name: Optional[SomeSubstitutionsType] = None,
+        namespace: Optional[SomeSubstitutionsType] = None,
+        parameters: Optional[SomeParameters] = None,
+        autostart: Optional[bool] = False,
+        remappings: Optional[SomeRemapRules] = None,
+        extra_arguments: Optional[SomeParameters] = None,
+        condition: Optional[Condition] = None,
```

Comment:
You can just forward these arguments to the ComposableNode class, only adding `autostart`, instead of redeclaring them here. This is also more robust to future changes as new arguments to `ComposableNode` will automatically be accepted here too.

You can use `kwargs` to do this. Take a look at how `LifecycleNode` is done. It duplicates a few parameters that it needs to use, which is fine, but pass the rest along entirely in `kwargs`.

Reply:
OK, done!